hybris/common/hooks.c

```diff
@@ -1591,8 +1592,9 @@ static int _hybris_hook_fsetpos(FILE *fp, const bionic_fpos_t *pos)
 
     fpos_t my_fpos;
     my_fpos.__pos = *pos;
-    memset(&my_fpos.__state, 0, sizeof(mbstate_t));
-    mbsinit(&my_fpos.__state);
+    if (!mbsinit(&my_fpos.__state)) {
```

Comment:
I think this should be just memset(&my_fpos.__state, 0, sizeof(mbstate_t)); without the entire mbsinit function which is used to check whether the variable is initialized but in this state we know it's uninitialized since we just created it.